Documentation/Home/_OverviewInclude.rst.txt

```diff
@@ -21,7 +21,7 @@ Getting Started with TYPO3
         *  :ref:`Installation Guide <t3start:installation_index>` details
            the entire installation process.
 
-        *  :ref:`Setup Guide <t3start:setup>` guides you through the next steps
+        *  :ref:`Setup Guide <t3start:first-project-setup>` guides you through the next steps
```

Comment:
The links link to version 12, I thought we want to link to 13? Then it has to be :ref:`Setup Guide <t3start/13:first-project-setup>`

Reply:
It will work after the switch in the guides